cfgov/paying_for_college/jinja2/paying-for-college/college-cost-blocks/macros/financial-item-expandable.html

```diff
@@ -43,16 +43,10 @@
             <span class="financial-item_value" {{ value.data_attribute | safe if value.data_attribute else '' }} >
                {{ value.value }}
             </span>
-            <span class="o-expandable_cue o-expandable_cue-open">
-                <span class="u-visually-hidden-on-mobile">
-                    {{ 'Edit' if value.is_editable else 'Show' }}
-                </span>
+            <span class="o-expandable_cue-open" aria-label="{{ 'Edit' if value.is_editable else 'Show' }}">
                 {{ svg_icon('edit') if value.is_editable else svg_icon('plus-round') }}
             </span>
-            <span class="o-expandable_cue o-expandable_cue-close">
-                <span class="u-visually-hidden-on-mobile">
-                    {{ 'Save' if value.is_editable else 'Hide' }}
-                </span>
+            <span class="o-expandable_cue-close" aria-label="{{ 'Save' if value.is_editable else 'Hide' }}">
```

Comment:
Same comment as above:

```suggestion
            <span class="o-expandable_cue-close" aria-label="{{ _('Save') if value.is_editable else _('Hide') }}">
```